learn/getting_started/quick_start.md

```diff
@@ -133,9 +133,9 @@ MeiliSearch **response**:
 
 [API references](/reference/api/search.md)
 
-### Web interface
+### Search preview
 
-We also deliver an out-of-the-box [web interface](/reference/features/web_interface.md) in which you can test MeiliSearch interactively.
+We also deliver an out-of-the-box [search preview interface](/learn/what_is_meilisearch/search_preview.md) in which you can test MeiliSearch interactively.
```

Comment:
Do we refer to it as the "search preview interface" as well?

Reply:
I think we should drop the `interface` here.

Reply:
I think you're probably right it can be dropped here, but I did use `interface` in a few spots, when `search preview` alone felt too abstract.

I'm wondering if that indicates a problem with our choice.

Reply:
Doesn’t preview alone make sense? 🤔

Reply:
Yeah, I'm probably overthinking it. I kept wondering "will people know what to expect when we say 'search preview'?"